app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,119 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS and SCSS file within this directory, lib/assets/stylesheets, vendor/assets/stylesheets,
+ * or any plugin's vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any styles
+ * defined in the other CSS/SCSS files in this directory. It is generally better to create a new
+ * file per style scope.
+ *
+ *= require_tree .
+ *= require_self
+ */
+
+html, body {
+ margin: 0;
+ padding: 0;
+ height: 100%;
+ font-family: 'Helvetica', sans-serif;
+}
+
```

Comment:
You're generally going to want to avoid putting any styling in `application.css`. It technically works to put CSS here, but this file's job is mainly to load and compile the rest of the files.